source/_static/clipboard-driver.js

```diff
@@ -0,0 +1,44 @@
+
+//Grab all elements with highlight-shell class - these will contain all shell commands
+var highlight_shell_elements = document.getElementsByClassName("highlight-shell");
+
+//Loop through elements with highlight-shell class and add clipboard button to each block
+for (x = 0; x < highlight_shell_elements.length; x++)
+{
+  //Create button element
+  var clipboard_button = document.createElement("BUTTON");
+
+  //Create span element and add glyphicon glyphicon-copy classes to span element
+  //Append span element as child of clipboard button
+  //Add clipboard-btn class to the clipboard button element
+  var span_element = document.createElement("SPAN");
+  span_element.setAttribute('class', 'glyphicon glyphicon-copy');
```

Comment:
Can you stick with one style of quotes? Either single (`'`) or double are fine (`"`), just as long as we stick with one in the file.